client/src/components/Support Page/SupportPage.js

```diff
@@ -3,7 +3,7 @@ import { Container, Grid, Tab, AppBar } from "@material-ui/core";
 import { makeStyles } from "@material-ui/core/styles";
 import Tabs from "./Tabs/Tabs";
 import { Route, Switch, Redirect } from "react-router-dom";
-
+import { Box } from "@material-ui/core";
```

Comment:
The Box component is imported but not used anywhere. If it is not used, please remove the import

Reply:
Done